recipes/vcf2hapmap/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/vcf2hapmap_conda
+  license: Artistic-2.0
+  summary: Convert VCF file to Hapmap file
+
+build:
+  detect_binary_files_with_prefix: true
+  number: 0
+  skip: False
+
+requirements:
+  run:
+    - perl
+
+package:
+  name: vcf2hapmap
+  version: 1.0.0
+
+source:
+  url: https://github.com/avengerpb/vcf2hapmap_conda/archive/1.0.tar.gz
```

Comment:
Hi, I want my recipe for this tool can be accessed more easily and more secured.